### PR TITLE
Update FAQ

### PR DESCRIPTION
**Pull request by puneetse**
This PR add 3 questions to the FAQ and revises top-down readability.

Closes https://github.com/clearlinux/clear-linux-documentation/issues/1059

### CODE REVIEW

File: source/FAQ/index.rst [hunk 1]
```diff
@@ -50,8 +67,9 @@ helps the |CL| team proactively identify and resolve bugs. See the
 What is the default firewall?
 =============================
 
-|CL| packages :command:`iptables` as a bundle, however, there are no default
-firewall rules. All network traffic is allowed by default.
+|CL| packages :command:`iptables` and :command:`firewalld` as optional
+bundles, however, there are no default firewall rules. All network traffic is
+allowed by default.
```
Comment (mvincerx) [inline source/FAQ/index.rst]: Ln 81.
Remove Ln 81: "Files under `:file: /etc/` ... and replace with:
|CL| ships with a mostly empty `/etc`, leaving configuration to the user. Read more about `user areas`_.

Add hyperlink at bottom: 
`.. _user areas: https://docs.01.org/clearlinux/latest/guides/clear/stateless.html#user-areas`

File: source/FAQ/index.rst [hunk 2]
```diff
@@ -68,13 +86,24 @@ https://clearlinux.org/news-blogs/where-etcfstab-clear-linux
...
-Software packages
-*****************
+Does it use the Intel Compiler (icc)?
+=====================================
+
+No. |CL| uses open source compilers: :command:`gcc` and :command:`clang`. |CL|
+does not compile any packages with :command:`icc`. 
+
+For a more detailed explanation, see this discussion on the community forum:
+https://community.clearlinux.org/t/does-clear-linux-os-use-the-intel-compiler-icc-tl-nope/
```
Comment (mvincerx) [inline source/FAQ/index.rst]: Embed hyperlink.  on Ln 95:
... see `this discussion`_ on the community forum. 
Add hyperlink at bottom of document: 
`.. _this discussion: https://community.clearlinux.org/t/does-clear-linux-os-use-the-intel-compiler-icc-tl-nope/`

File: source/FAQ/index.rst (hunk 1 above)
Comment (mvincerx) [reply source/FAQ/index.rst]: Ln 83-85, also please embed hyperlink. 
See `this blog post`_ explaining how to customize `/etc/fstab`.
Add hyperlink at bottom: 
.. _this blog post: https://clearlinux.org/news-blogs/where-etcfstab-clear-linux

File: source/FAQ/index.rst [hunk 3]
```diff
@@ -86,15 +115,28 @@ bundles whenever possible.
...
-dependencies at OS build time.
+The |CL| team wants software *installation* and *updates* to be as efficient
+and error free as possible. |CL| packages software differently and uses a
+novel updater to solve some of the classic problems with how the software
+packages are on Linux. 
 
-Individual RPMs can sometimes be manually installed on a |CL| system with the
-right tools, but that is not the intended use case.
+For a more detailed explanation, see this discussion on our community forum:
+https://community.clearlinux.org/t/why-does-clearlinux-use-swupd-and-not-apt-deb-rpm/
```
Comment (mvincerx) [inline source/FAQ/index.rst]: Ln 138-139: Embed hyperlink .
For a more detailed explanation, see this `discussion on swupd`_ in our community forum.

Add hyperlink at bottom: 
.. _discussion on swupd: https://community.clearlinux.org/t/why-does-clearlinux-use-swupd-and-not-apt-deb-rpm/3284

**Comment by puneetse** [comment]
@mvincerx Thanks. I pushed an update. For FAQ maintainability I prefer hyperlinks in-line. Yes, if more editorial changes are required please open a new PR.

**Comment by mvincerx** [comment]
LGTM. Thanks for the talk.

**Pull request merged**